docs/auth/index.md

```diff
@@ -10,6 +10,13 @@ configure Backstage to have any number of authentication providers, but only
 one of these will typically be used for sign-in, with the rest being used to provide
 access external resources.
 
+> NOTE: Identity management and the Sign-In page in Backstage is NOT a method for blocking
+> access for unauthorized users. The identity system only serves to provide a personalized
+> experience and access to a Backstage Identity Token, which can be passed to backend plugins.
+> This also means that your Backstage backend APIs are by default unauthenticated.
+> Thus, if your Backstage instance is exposed to the Internet, anyone can access
+> information in the Backstage. You can learn more [here](../overview/threat-model#integrator-responsibilities).
```

Comment:
Looks like this link is failing the build: `Unable to reach ../overview/threat-model from root or microsite/static/, linked from docs/auth/index.md`

Reply:
right, missed the file extension